Uppsala/UppEt45.xml

```diff
@@ -0,0 +1,381 @@
+                                ኪርምያል፡ አፍዳኮዳዮዳ፡ ሕልማና፡ ያሰርስ፡ ብንያክኤል፡ ሶርስሳብ፡ ሮስሳብ፡ እርዳድ፡ 
+                                ለምንያም፡ አልሚስ፡ ጢራጢብንዮስ፡ ስምዓኒ፡ ለአመትከ፡ 
+                                <persName ref="PRS14625WalattaMaryam" role="owner"><hi rend="rubric">ወለተ፡</hi>
+                                <subst><del rend="effaced" unit="chars" quantity="7"/><add>ማርያም፡</add></subst></persName>
+                                ወትቤ፡ እግዝእትነ፡ በነገረ፡ ዕብራይስጢ፡ ሶፎሶሮያሮስቦ፡ ወክርሜል<supplied reason="omitted">፡</supplied>
+                            </explicit>
+                        </msItem>
+                      
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>940</height>
+                                        <width>240</width>
+                                    </dimensions>
+                                    <note>Composed of one strip.</note>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="101">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Able hand of the 19th century (<foreign xml:lang="de">von einer habilen Hand des 19. Jahrhunderts</foreign>, 
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">137</citedRange></bibl>).
+                                    </desc>
+                                <date notBefore="1800" notAfter="1899">19th century.</date>
+                            </handNote>
+                        </handDesc>
+
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <desc>Coloured <term key="Magic">magic picture</term> of 
+                                    a man with round eyes showing the onlooker the palms of his hands.
+                                    From the top of his head, two snakes emerges, one reaching towards the left
+                                    and the other to the right.
+                                    The picture is placed in a field of 42 cm height, which it almost fills. 
```

Comment:
really 42 cm, not mm?
Does the catalogue provide any information on where in the scroll the picture is located?
The dimension can also be encoded, see guidelines:
  ```
<decoNote xml:id="d2" type="drawing" corresp="#pic2">
                            <desc>Monochrome drawing of a magic square, possibly a <ref type="authFile" corresp="AT1140SealSol"></ref>, but without face.</desc>
                            <dimensions unit="mm">
                                <height>75</height>
                                <width>46</width>
                            </dimensions>
                        </decoNote>
```

Reply:
Yes, cm! :) 

I technically don't have the measurements of the image, only of the field where it appears…

<img width="170" alt="Skärmavbild 2025-01-07 kl  09 08 00" src="https://github.com/user-attachments/assets/3e964615-1f0c-4117-a680-6122718f076e" />